packages/dara-components/changelog.md

```diff
@@ -2,7 +2,7 @@
 title: Changelog
 ---
 
-## NEXT
+## 1.10.3
 
 -   Fixed an issue where `onchange` would not fire for `Input` and `Textarea` components in some cases
```

Comment:
Should we add an entry from the dara-ui changelog here?

Reply:
Yes, good catch!